UPDATING.md

```diff
@@ -29,6 +29,8 @@ assists people when migrating to a new version.
 
 ### Breaking Changes
 
+- [26349](https://github.com/apache/superset/issues/26349): Removes the deprecated `DASHBOARD_CACHE` feature flag.
```

Comment:
What are the consequences of removing the cache? Is there merit in expanding why this was removed?

Reply:
I think this feature never worked correctly. It was Airbnb who introduced it and quickly disabled it because of problems reported by users while using the dashboards.